### ISSUE TITLE
can't select specific test case in v0.31.1

### ISSUE THREAD

**Issue by LuisAfonso95** [ThrowTheSwitch/Ceedling#832]
Hello, I am unsure if this is a bug or just missing from current release but,

I'm using ceedling v0.31.1 installed through gem.

I've tried adding to the project.yml
```ruby
:test_runner:
  :cmdline_args: TRUE
``` 

and 
 
```ruby
:defines:
  # in order to add common defines:
  #  1) remove the trailing [] from the :common: section
  #  2) add entries to the :common: section (e.g. :test: has TEST defined)
  :common: &common_defines 
    - UNITY_USE_COMMAND_LINE_ARGS
``` 

Then I call 
`ceedling test:FILE --test_case=<test case name>`

But it just runs all tests in FILE. No matter what I write in test case or no matter which argument I use it just seems too ignore it all together.

**Comment by Letme** [comment]
You mean that worked in some previous version but it doesn't work anymore?

**Comment by LuisAfonso95** [comment]
I have never tried using ceedling in a previous version

**Comment by Letme** [comment]
So this is feature request, to execute a single test case out of a test file?

**Comment by LuisAfonso95** [comment]
The feature is mentioned in [the docs](https://github.com/ThrowTheSwitch/Ceedling/blob/master/docs/CeedlingPacket.md) but now I realize this is not in the v0.31.1 docs 
I guess I'll need to try a release candidate

**Comment by Letme** [comment]
Sorry, I was not aware of the feature.

`FILE` I assume is without path, without `test_` and `.c`, while testcase is also without `test_` prefix?

**Comment by LuisAfonso95** [comment]
yes I've tried multiple combinations. FILE always been without the path and just the module name otherwise it doesn't even run.

the testcase I've tried multiple combinations. With test_, without test_ and other substrings.

It always runs the entire FILE test cases

**Comment by JuPrgn** [comment]
@Letme this is related to #831
